frontend/webpack.config.ts

```diff
@@ -132,25 +112,6 @@ const config: Configuration = {
           getModuleData: () => getActivePluginsModuleData(staticPluginPackages),
         },
       },
-      {
-        test: /\.js$/,
-        include: /node_modules\/@patternfly-4\//,
-        loader: 'babel-loader',
-        options: {
-          plugins: [
-            [
-              'transform-imports',
```

Comment:
this was the only usage of `babel-plugin-transform-imports` so it should be safe to remove from package.json

Reply:
This was to ensure that code under `node_modules/@patternfly-4` directory is referencing other PF v4 packages (instead of PF v5) using the `@patternfly-4` aliased scope.

I think we may need this again once we upgrade to PF v6 and keep the PF v5 core package with CSS styles.

I'm OK with doing the removal as @logonoff mentioned now and re-evaluate later on.